docs/kb/semgrep-code/collect-cli-logs.md

```diff
@@ -0,0 +1,77 @@
+---
+tags:
+  - Semgrep CLI
+  - Logs
+description: How to collect logs when running Semgrep on the command line.
+---
+
+# How to collect logs when running Semgrep in CLI
+
+When troubleshooting Semgrep scans on the command line (CLI), collecting and sharing logs can be extremely helpful. By default, Semgrep prints findings from a scan to `stdout`, and other messages, including scan details and progress, to `stderr`. For troubleshooting, it's most often useful to provide both.
```

Comment:
```suggestion
When troubleshooting Semgrep scans on the command line interface (CLI), collecting and sharing logs can be extremely helpful. By default, Semgrep prints findings from a scan to `stdout`, and other messages, including scan details and progress, to `stderr`. For troubleshooting, it's most often useful to provide both.
```
Just to match up the acronyms

Reply:
Yeah, I thought this sounded weird, but I also feel like "scans on the command line interface" sounds weird. That said I think it's better than the option I chose. :)